layouts/partials/help.html

```diff
@@ -3,25 +3,33 @@
     <div class="container">
         <div class="row">
             <div class="col-lg-12 text-center">
-                <h2 class="section-heading">{{ with .Site.Params.help.title }}{{ . }}{{ end }}</h2>
-                <h3 class="section-subheading text-muted">{{ with .Site.Params.help.subtitle }}{{ . | markdownify }}{{ end }}</h3>
```

Comment:
I forget if it was this line or the one above where I'd put the date for the current event in the toml param.